docs/development/troubleshooting.md

```diff
@@ -257,3 +257,10 @@ and relying on DJL's `NDManager` to efficiently GC the memory. You can do that b
 ```
 -Dai.djl.disable_close_resource_on_finalize=true
 ```
+
+## 11. Special character issues (PaddlePaddle)
+
+Please remember to not include special characters in your data/model path. Like "数据" or some non-asicii character.
+
+![Special Char](img/cn_char.jpg)
```

Comment:
We should avoid use image, just use text version.
The image contains user name, we should at least use different one.